arrayfire/array.py

```diff
@@ -1185,20 +1185,35 @@ def to_list(self, row_major=False):
         ct_array, shape = self.to_ctype(row_major, True)
         return _ctype_to_lists(ct_array, len(shape) - 1, shape)
 
-    def __repr__(self):
+    def to_string(self):
```

Comment:
* `to_string` renamed to `__str__`
* added `get/set_display_dims_limit` functions (default: `None`)
* `__str__` display_dims_limit aware, if out of limit or is PyCharm detected it simply calls `__repr__`